src/sidebar/components/test/annotation-thread-test.js

```diff
@@ -81,8 +81,7 @@ describe('annotationThread', function() {
       },
     });
     const pageObject = new PageObject(element);
-    assert.equal(pageObject.annotations().length, 1);
-    assert.isTrue(pageObject.isHidden(pageObject.annotations()[0]));
+    assert.equal(pageObject.annotations().length, 0);
```

Comment:
Note this change. I believe this to be OK: instead of rendering "hidden" annotation (threads) if `thread.visible` is `false`, we now no longer render at all. I believe this change to be OK because hidden threads no longer need to be in the DOM at all (no remaining logical reason for this that I know of). But please review!